src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -186,6 +239,9 @@
 (def default-inventory-lock-timeout-seconds
   60)
 
+(def default-auto-renewal-cert-ttl
+  "60d")
```

Comment:
We should do the math here, and call out what it is, `default-auto-renewal-cert-ttl-seconds`.  I have something similar in my open PR, I believe.